src/test/java/io/supertokens/test/TestingProcessManager.java

```diff
@@ -145,4 +150,23 @@ EventAndException checkOrWaitForEvent(PROCESS_STATE state, long timeToWaitMS)
             return e;
         }
     }
+
+    /**
+     * Utility function to wrap tests with, as they require TestingProcess
+     */
+    public static void withProcess(ProcessConsumer consumer) throws Exception {
```

Comment:
- Maybe you should add an optional param (via overloading) which will allow for not MySQL based dbs (cause session tests can work with mongodb as well).
- This would also need to go in each of the plugins, as they have their own `TestingProcessManager` class.

Reply:
I moved that to a test, since that was common, but not related to process itself (although it depends on a process). Maybe we can just supply some common static function to check that (or base `SupertokensTest` class that would take care of such repetitive tasks).